pages/models/generations/SM1/GENROU/index.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: page
+title: GENROU
+tags: ["#110","Synchronous machine", "RMS", "EMT", "Phasorial", "Rotor", "Stator", "Dynawo", "Opensource", "GENROU", "GENSAL"]
+date: 10/04/2024 
+last-updated: 23/04/2024
+---
+
+# Salient-pole Synchronous Machine
+
+## Context
+
+As explained in the general Synchronous Machine model, the particular systems that can be derived by making some additional assumptions yield systems of equations that can be easily solved computationally. The GENROU model in *abc* reference frame is one of the simplest models, considering the uniformity of the air gap between the stator and rotor to be able to simplify the equations. 
+
+## Model use, assumptions, validity domain and limitations
+
+The model can be used to perform dynamic simulations of round-rotor synchronous machine in *abc* reference frame, aswell as performing steady-state analysis using the equivalent circuit obtained.
+
+The additional assumptions with respect to the base model made in this model are:
+
+- The air gap between the stator and rotor is uniform.
+- The three-phase system is balanced.
+
+It cannot be used to simulate machines with a non-uniform air gap such as the salient-pole synchronous machine. In fact, the round-rotor model is a particular case of the salient-pole model.
+
+## Model Description
+
+### Parameters
```

Comment:
i assume you will complete this part, as you did for gensal?

Reply:
Model deleted